### ISSUE TITLE
About failed to read some frames

### ISSUE THREAD

**Issue by beihaidao** [facebook/C3D#301]
in C3D-v11
when run :  sh finetuning_ucf101.sh 
then i will get same error like this :
I0828 09:16:08.501374 19888 sgd_solver.cpp:105] Iteration 400, lr = 0.001
E0828 09:16:13.051661 19895 image_io.cpp:339] Could not open or find file /home/xxx/ucf_frame/v_TableTennisShot_g10_c02//000207.jpg
E0828 09:16:15.679214 19895 image_io.cpp:339] Could not open or find file /home/xxx/ucf_frame/v_BlowingCandles_g21_c04//000063.jpg
E0828 09:17:35.677505 19895 image_io.cpp:339] Could not open or find file /home/xxx/ucf_frame/v_Basketball_g16_c06//000315.jpg
I0828 09:19:06.901814 19888 solver.cpp:219] Iteration 600 (1.12108 iter/s, 178.4s/200 iters), loss = 2.08726
I0828 09:19:06.902577 19888 solver.cpp:238]     Train net output #0: loss = 2.08726 (* 1 = 2.08726 loss)
I0828 09:19:06.902587 19888 sgd_solver.cpp:105] Iteration 600, lr = 0.001
## E0828 09:19:20.292134 19895 image_io.cpp:339] Could not open or find file /home/xxx/ucf_frame/v_BalanceBeam_g10_c01//000079.jpg
In   train_01.slt, i see : v_BalanceBeam_g10_c01/ 65 4
but    /home/xxx/ucf_frame/v_BalanceBeam_g10_c01// ## 000078.jpg . is the last frame.  ## but 65+16=81 >78
so how to  ## solve the problem <start frame>+16 >last frame ?

**Comment by dasha50** [comment]
I just delete the last insufficient frames.

**Comment by beihaidao** [comment]
i have solve the problem by 
extract feature  again like this:
ffmpeg -i "/path/to/video" "/path/to/frm/dir/%06d.jpg"
see also https://github.com/facebook/C3D/issues/113
